### DISCUSSION TITLE
BoundingBox#intersect removed

### DISCUSSION THREAD

**Discussion by yusshu** [Minestom/Minestom#827] [Help] [answered]
What is the current equivalent for BoundingBox#intersect(double, double, double, double, double, double)?
Removed in this commit https://github.com/Minestom/Minestom/commit/6891a530f5ddd4aab31590f21b5a64d029401f2a#diff-904d08063fcc098ea91623e46dd03cad5c0bfef191242404727b31a00cfd9311L154

XY: Detecting interactions of players on *custom* entities

**Comment by iam4722202468** [comment]
`BoundingBox#intersectEntity(<this entity position>, <other entity>)` should work

Update: The functionality of the removed method was to check if a line segment intersected a bounding, not check if two bounding boxes intersect

**Reply by yusshu** [reply]
Solved, I'm using `Entity#getLineOfSightEntity`, thank you anyways

**Comment by yusshu** [comment, marked as answer]
`Entity#getLineOfSightEntity` works for me
